docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/kubernetes.md

```diff
@@ -0,0 +1,69 @@
+
+## Hardware requirements
+
+A Kubernetes node is expected to meet the following requirements: 
+
+- 5 cores (minimum) or 8 cores (recommended)
+- 15 GB RAM (minimum)
+- 250 GB SSD disk (minimum)
+- 64-bit CPU architecture
+
+## Prepare the cluster
+
+The YugabyteDB Anywhere Helm chart has been tested using the following software versions:
+
+- Kubernetes 1.22
+- Helm 3.10
+
+
+Before installing YugabyteDB Anywhere, verify that you have the following:
+
+- A Kubernetes cluster with nodes configured according to the [hardware requirements](#hardware-requirements).
+- A client environment with the kubectl and Helm command-line tools configured with a service account or user that has admin access to a  single namespace on the subject Kubernetes cluster.
+- A Kubernetes secret obtained from [Yugabyte](https://www.yugabyte.com/platform/#request-trial-form).
+
+In addition, ensure the following:
+
+- The nodes can pull container images from the [quay.io](https://quay.io/) container registry. If the nodes cannot do this, you need to prepare these images in your internal registry by following instructions provided in [Pull and push YugabyteDB Docker images to private container registry](../../prepare-environment/kubernetes#pull-and-push-yugabytedb-docker-images-to-private-container-registry). 
+- Core dumps are enabled and configured on the underlying Kubernetes node. For details, see [Specify ulimit and remember the location of core dumps](../../prepare-environment/kubernetes#specify-ulimit-and-remember-the-location-of-core-dumps).
+- You have the kube-state-metrics add-on version 1.9 in your Kubernetes cluster. For more information, see [Install kube-state-metrics](../../prepare-environment/kubernetes#install-kube-state-metrics).
+- A load balancer controller is available in your Kubernetes cluster.
+- A StorageClass is available with the SSD and `WaitForFirstConsumer` preferably set to `allowVolumeExpansion`. For more information, see [Configure storage class](../../prepare-environment/kubernetes/#configure-storage-class).
+
+## Multi cluster Kubernetes environment
+
+When you want to create a multi-region YugabyteDB universe or two universes from different regions replicating using xCluster, you need to use one Kubernetes cluster per region. In the common case, following are the prerequisites for YugabyteDB Anywhere to work across multiple Kubernetes clusters:
+
+- Pod IP address connectivity should be present between the clusters. Each pod and service should have an unique IP address across the clusters (non-overlapping addresses).
+- There should be DNS connectivity between the clusters. ClusterIP and headless service FQDNs (including the individual pod FQDNs) exposed in one Kubernetes cluster should be resolvable in all the other Kubernetes clusters.
```

Comment:
Cluster IP?

Reply:
ClusterIP is a type of service in Kubernetes i.e. it is a keyword.